src/react/components/pages/homepage/homePage.tsx

```diff
@@ -156,9 +156,15 @@ export default class HomePage extends React.Component<IHomePageProps, IHomePageS
     }
 
     private loadSelectedProject = async (project: IProject, sharedToken?: {}) => {
-        const loadedProject = await this.props.actions.loadProject(project, sharedToken);
-        if (loadedProject !== null) {
-            this.props.history.push(`/projects/${project.id}/edit`);
+        try {
+            const loadedProject = await this.props.actions.loadProject(project, sharedToken);
+            if (loadedProject !== null) {
+                this.props.history.push(`/projects/${project.id}/edit`);
+            }
+        } catch (error) {
+            if (error instanceof AppError && error.errorCode === ErrorCode.SecurityTokenNotFound) {
+                toast.error(strings.errors.securityTokenNotFound.message, { autoClose: 3000 });
```

Comment:
I think we need increase the autoClose for 5 seconds - user have time to read it, or even turn off autoClose.